### PR TITLE
Fix dynamic WF marshalling

### PR DESCRIPTION
**Pull request by EngHabu**

### CODE REVIEW

**Comment by EngHabu** [comment]
PTAL @kumare3

File: pkg/controller/nodes/dynamic/handler.go [hunk 1]
```diff
@@ -288,7 +288,7 @@ func (e dynamicNodeHandler) buildFlyteWorkflow(ctx context.Context, w v1alpha1.E
 		return nil, true, err
 	}
```
Comment (kumare3) [inline pkg/controller/nodes/dynamic/handler.go]: we should have a test for this?

**Comment by EngHabu** [comment]
🚀 🙏

File: pkg/controller/nodes/dynamic/handler.go [hunk 2]
```diff
@@ -288,7 +288,7 @@ func (e dynamicNodeHandler) buildFlyteWorkflow(ctx context.Context, w v1alpha1.E
...
 
-	subwf, err := k8s.BuildFlyteWorkflow(closure, nil, nil, "")
+	subwf, err := k8s.BuildFlyteWorkflow(closure, &core.LiteralMap{}, nil, "")
```
Comment (kumare3) [inline pkg/controller/nodes/dynamic/handler.go]: we dont need to do this, i fixed it in the marshaler itself

**Comment by kumare3** [comment]
👍

**Pull request merged**